ui/src/widgets/ui-form/UIForm.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <label v-if="label" class="nrdb-ui-form-label">{{ label }}</label>
-    <v-form ref="form" v-model="isValid" validate-on="blur" @submit.prevent="onSubmit">
+    <v-form ref="form" v-model="isValid" :disabled="!state.enabled" validate-on="input" @submit.prevent="onSubmit">
```

Comment:
validation on input can be quite annoying, as in the case of an `e-mail` it validates as "error" the first character you put in, until you type an `@`

Any particular reason you changed this?

Reply:
Was a way to fix #935, but didn't try with email type, my test form only had `number` and `text`. Will check if there is a better option

Reply:
a `validate()` in a `nextTick()` when a new message is received should do the trick there

Reply:
In this case there is no message, the user is filling the `form` and if it goes directly from a text input to the validate button, and it is not `enabled`, two clicks are required, the first click trigger the `onBlur` event and enables the submit button for the second click.

Reply:
@joepavitt just tried this, and even with the UI-Form set to `email` the validation seems OK, in reality it is not even validating that is a valid email (with `@`) as long as write any character the validation is "ok", it is just checking that the field cannot be empty.